tests/pages/firefox/sync.py

```diff
@@ -13,6 +13,7 @@ class FirefoxSyncPage(FirefoxBasePage):
 
     _primary_download_button_locator = (By.ID, 'download-button-desktop-release')
     _android_download_button_locator = (By.ID, 'cta-android-footer')
+    _ios_download_button_locator = (By.ID, 'cta-ios-footer')
```

Comment:
We could also rename these locators to something a bit more semantic, now that we're using store badges instead of download buttons. Something like `_play_store_button_locator` and `_app_store_button_locator`?